File: SoloLearn_CodeCoach/Easy/GothamCity/GothamCity.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;
    if (n < 5) {
        cout << "I got this!" << endl;
    } else if (n < 10) {
        cout << "Help me Batman" << endl;
    } else {
        cout << "Good Luck out there!" << endl;
    }

    return 0;
}

File: SoloLearn_CodeCoach/Easy/GothamCity/GothamCity.java
import java.util.Scanner;

public class GothamCity {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int numberOfCriminals = input.nextInt();

        if (numberOfCriminals < 5) {
            System.out.println("I got this!");
        } else if (numberOfCriminals <= 10) {
            System.out.println("Help me Batman");
        } else {
            System.out.println("Good Luck out there!");
        }
    }
}
